### ISSUE TITLE
Add new leveldb version for windows

### ISSUE THREAD

**Issue by chriseth** [ethereum/webthree-umbrella#468]
The current version of leveldb we use (leveldb v 1.02) on windows seems to mess with the filesystem and locks. We should find a better replacement for it, @winsvega is already working on that.
The dependency will be in github.com/ethereum/leveldb and files in https://github.com/ethereum/cpp-dependencies should be updated accordingly.

Task breakdown:
- [x] find, modify and test new database
- [x] update cpp-dependencies
- [x] update windows build machine

**Comment by winsvega** [comment]
this PR updates windows version of bitcoin-fork 
leveldb  (win64) 1.02 -> 1.18
https://github.com/ethereum/leveldb/pull/4

the binaries are here
https://drive.google.com/file/d/0B3Yr0ZZNlHv2NjdKam1TRzV1Uzg/view?usp=sharing

tested on win7. once db is opened by new version of leveldb, it could not be opened by previous versions. (waiting for bob to test it on win10)

need to replace the binaries here among with loading ext dep sctipts on windows 
https://build.ethdev.com/builds/windows-precompiled/

**Comment by chriseth** [comment]
Please add the library in UseEth.cmake

**Comment by chriseth** [comment]
Uploaded the binary. @winsvega could you list what else needs to be changed? What do we have to do on the build machine?

**Comment by chriseth** [comment]
Upgraded the build machine. testing still pending (happy for anyone else to pick up)

**Comment by bobsummerwill** [comment]
What testing is required?
Just running getstuff.bat and verifying that the results are OK?
Or verifying the binaries generated the ZIP from the build machine?

**Comment by chriseth** [comment]
I think we should verify the binaries, there should be fresh develop binaries as of 6 hours ago

**Comment by bobsummerwill** [comment]
OK.    I think we're done!
- I just verified that getstuff.bat is now working.   It is.
- I just unzipped win_eth.zip and ran eth successfully for a bit.
- I just uninstalled my previous setup and ran the new installer and successfully opened AlethZero and Mix, and then ran eth.exe for a bit.

All this on a Surface Pro 3 running Windows 10 Anniversary Edition preview.

So I'll close this, and we can reopen if we spot any more work later.

Good job everyone, and especially @winsvega !
